_posts/2018-07-10-react-native-wdio.md

```diff
@@ -0,0 +1,358 @@
+---
+layout: post
+title: "React Native Application UI testing using WebDriverIO and Appium"
+date: 2018-07-09 22:58:15 -0800
+cover: /assets/images/cover-wdio.png
+excerpt: We recently adopted WebDriverIO based UI testing for our React Native application. Benefits of using WebDriverIO include allowing us to write UI tests just as we wrote tests for the web. The [synchronous nature of WebDriverIO](http://webdriver.io/guide/getstarted/v4.html#It%E2%80%99s-all-synchronous) helps us write clean UI tests with PageObject models. WebDriverIO configuration allows us to plugin SauceLabs Emulators or Real Devices for cloud-based testing.
```

Comment:
Remove the hyperlink from the excerpt. The front matter isn't converted like Markdown is. Also, if you see anything that could be trimmed down, every character helps! Here is a screenshot of what that currently looks like on the blog index page:

![screen shot 2018-07-10 at 9 41 48 am](https://user-images.githubusercontent.com/1934719/42524553-b1dce78c-8425-11e8-81af-35d91b2f484c.png)